packages/theme/src/themes.ts

```diff
@@ -48,6 +48,10 @@ export const lightTheme = createTheme({
       text: '#FFFFFF',
       link: '#000000',
     },
+    code: {
+      background: '#333333',
+      text: '#FFFFFF',
+    },
```

Comment:
I wonder if we can go with a similar styling like the code conponent? https://backstage.io/storybook/?path=/story/data-display-codesnippet--default

Reply:
Like your idea! Added the CodeSnipped component as a custom renderer for code blocks. 

![Bildschirmfoto 2020-10-30 um 14 25 11](https://user-images.githubusercontent.com/61695677/97710261-d209db80-1abb-11eb-91dc-3410a85eb45a.png)
![Bildschirmfoto 2020-10-30 um 14 25 16](https://user-images.githubusercontent.com/61695677/97710264-d33b0880-1abb-11eb-8f4c-5cee235ed105.png)

Reply:
Looks great